src/test/java/gov/cabinetoffice/gap/applybackend/web/SubmissionControllerTest.java

```diff
@@ -257,855 +310,887 @@ class SubmissionControllerTest {
 
-        @Test
-        void testExportSingleSubmission_OdtService_throws_ForbiddenException() {
-            UUID submissionId = UUID.randomUUID();
-            HttpServletRequest mockRequest = new MockHttpServletRequest();
-            Submission mockSubmission = mock(Submission.class);
-            when(grantApplicantService.getEmailById(any(), any())).thenReturn("mock@example.com");
-            when(submissionService.getSubmissionById(any())).thenReturn(mockSubmission);
-            when(submissionService.getSubmissionExport(any(), any(), any()))
-                    .thenThrow(new ForbiddenException());
-
-            RuntimeException runtimeException = assertThrows(RuntimeException.class, () -> controllerUnderTest
-                    .exportSingleSubmission(submissionId, mockRequest));
-            assertTrue(runtimeException.getCause() instanceof ForbiddenException);
-            verify(submissionService, times(1)).getSubmissionExport(any(), any(), any());
-        }
+                final ResponseEntity<String> response = controllerUnderTest.applicationStatus(SUBMISSION_ID);
 
-        @Test
-        void testExportSingleSubmission_zipService_throws_IOException() throws Exception {
-            UUID submissionId = UUID.randomUUID();
-            HttpServletRequest mockRequest = new MockHttpServletRequest();
-            Submission mockSubmission = mock(Submission.class);
-
-            when(grantApplicantService.getEmailById(any(), any())).thenReturn("mock@example.com");
-            when(submissionService.getSubmissionById(any())).thenReturn(mockSubmission);
-            when(submissionService.getSubmissionExport(any(), any(), any()))
-                    .thenReturn(mock(OdfTextDocument.class));
-            when(zipService.createSubmissionZip(any(), any()))
-                    .thenThrow(new IOException("Error creating zip file"));
-
-            RuntimeException runtimeException = assertThrows(RuntimeException.class, () -> {
-                controllerUnderTest.exportSingleSubmission(submissionId, mockRequest);
-            });
-
-            assertTrue(runtimeException.getCause() instanceof IOException);
-            verify(submissionService, times(1)).getSubmissionExport(any(), any(), any());
-            verify(zipService, times(1)).createSubmissionZip(any(), any());
+                assertEquals(HttpStatus.UNAUTHORIZED, response.getStatusCode());
+            }
         }
 
+    }
 
-        @Test
-        void exportSingleSubmissionThrowsRuntimeError() {
-            UUID submissionID = UUID.randomUUID();
-            final MockHttpServletRequest request = new MockHttpServletRequest();
-            when(submissionService.getSubmissionExport(Mockito.any(), Mockito.anyString(), Mockito.anyString()))
-                    .thenThrow(new ForbiddenException());
 
-            assertThrows(RuntimeException.class, () -> controllerUnderTest.exportSingleSubmission(submissionID, request));
+    @Nested
```

Comment:
This is the test changed, the rest of the differences, are made by reorganising the test, and split them in 2 categories. one that uses the security context, and the other that doesnt